lib/combine_pdf/decrypt.rb

```diff
@@ -143,12 +140,16 @@ def decrypt_AES(encrypted, encrypted_id, encrypted_generation, _encrypted_filter
       object_key << 'sAlT'.force_encoding(Encoding::ASCII_8BIT)
       key_length = object_key.length < 16 ? object_key.length : 16
 
-      cipher = OpenSSL::Cipher.new("aes-#{key_length << 3}-cbc")
-      cipher.decrypt
-      cipher.key = Digest::MD5.digest(object_key)[(0...key_length)]
-      cipher.iv = encrypted[0..15]
-      cipher.padding = 0
-      cipher.update(encrypted[16..-1]) + cipher.final
+      begin
+        cipher = OpenSSL::Cipher.new("aes-#{key_length << 3}-cbc")
+        cipher.decrypt
+        cipher.key = Digest::MD5.digest(object_key)[(0...key_length)]
+        cipher.iv = encrypted[0..15]
+        cipher.padding = 0
+        cipher.update(encrypted[16..-1]) + cipher.final
+      rescue Exception => e
```

Comment:
Oh, I forgot to finish this part.

The reason why I had to catch exceptions here is that somehow, the code tried to decrypt data that was not encrypted.

Returning `encrypted` as-is when `cipher` throws `bad encryption` error was a quick and dirty fix. I left a mental note to investigate it further (or at least handle the exception properly and leave a note) but forgot about it.

I think `d8b1328b964ceb87cb6c83d0c7210bef3013c200` may have fixed that issue.